tools/update-dll-msvc.bat

```diff
@@ -117,18 +170,7 @@ if %msvc_ver% geq 11 (
 )
 
 if errorlevel 1 (
-    rem Build failed.
-
-    rem Try to delete old DLLs.
-    if exist lib\%vimproc_dllname%.old del lib\%vimproc_dllname%.old
-    if exist lib\%vimproc_dllname%     del lib\%vimproc_dllname%
-    rem If the DLL couldn't delete (may be it is in use), rename it.
-    if exist lib\%vimproc_dllname%     ren lib\%vimproc_dllname% %vimproc_dllname%.old
```

Comment:
Why you removed this block?
This intends that vimproc.dll will be updated even if the dll is in use.

Reply:
@k-takata , Previousely I think the errorlevel 1 are also reflect the result of build failed. Thank is why I change it to "make clean". Now I need to correct my understanding as you said. I will fix it at new commit.